server/pulp/server/async/manage_workers.py

```diff
@@ -18,11 +18,12 @@
 
 [Service]
 EnvironmentFile=%(environment_file)s
+Environment=PULP_MAX_TASKS_PER_CHILD=0
 User=apache
 WorkingDirectory=/var/run/pulp/
 ExecStart=/usr/bin/celery worker -n reserved_resource_worker-%(num)s@%%%%h -A pulp.server.async.app\
           -c 1 --events --umask 18 --pidfile=/var/run/pulp/reserved_resource_worker-%(num)s.pid\
-          --heartbeat-interval=30
+          --heartbeat-interval=30 --maxrequestsperchild=$PULP_MAX_TASKS_PER_CHILD
```

Comment:
Similar question here about this being set 100% of the time with a default value of 0.